src/view/services/command.manager.js

```diff
@@ -0,0 +1,12 @@
+export class CommandManager {
+	constructor(apply) {
+		this.apply = apply;
+	}
+
+	execute(commands) {
+		// First we need to get list of executable commands, cause execution of prev command can
```

Comment:
As I understood, `commandManager.execute` is triggered only from `onKeyDown` method of Shortcut instance. Why `enter` is triggered twice per single key press?
I think this approach may introduce new bugs in future.